.circleci/config.yml

```diff
@@ -22,21 +33,22 @@ jobs:
             . venv/bin/activate
             pip install --upgrade pip
             pip install --upgrade setuptools
-            cd ml-agents-envs && pip install -e .
-            cd ../ml-agents && pip install -e .
-            pip install pre-commit pytest-cov==2.6.1
-            cd ../gym-unity && pip install -e .
+            pushd ml-agents-envs && pip install -e . && popd
+            pushd ml-agents && pip install -e . && popd
+            pip install -r test_requirements.txt
+            pushd gym-unity && pip install -e . && popd
 
       - save_cache:
           paths:
             - ./venv
-          key: v1-dependencies-{{ checksum "ml-agents/setup.py" }}
+          key: v1-dependencies-py<< parameters.pyversion >>-{{ checksum "ml-agents/setup.py" }}
 
       - run:
           name: Run Tests for ml-agents and gym_unity
           command: |
             . venv/bin/activate
             mkdir test-reports
+            pip freeze > test-reports/pip_versions.txt
```

Comment:
Dump the installed package versions to an artifact; this should at least give us an idea of what versions we're running in testing.

Reply:
Can we add as a comment in the yml file?

Reply:
Done